cloud/azure/modules/app/locals.tf

```diff
@@ -48,11 +48,12 @@ locals {
 
     SECRET_KEY = data.azurerm_key_vault_secret.app_secret_key.value
 
-    ADFS_SECRET        = data.azurerm_key_vault_secret.adfs_secret.value
-    ADFS_CLIENT_ID     = data.azurerm_key_vault_secret.adfs_client_id.value
-    ADFS_DISCOVERY_URI = data.azurerm_key_vault_secret.adfs_discovery_uri.value
-
-    CIVIFORM_APPLICANT_IDP = var.civiform_applicant_idp
+    AD_GROUPS_ATTRIBUTE_NAME = "groups"
```

Comment:
suggestion (non-blocking): make this a variable with a default value instead of hard-coding